lib/sort_by_length.rb

```diff
@@ -1,7 +1,25 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)
+# Space complexity: O(n)
+# selection sort attempt:
+
 def sort_by_length(my_sentence)
```

Comment:
Nice insertion sort.  However the time complexity is O(n^2), take a look at the nested loops.